src/PROPOSAL/scattering/stochastic_deflection/nuclearInteraction/NaivNuclearInteraction.cxx

```diff
@@ -0,0 +1,33 @@
+#include "PROPOSAL/scattering/stochastic_deflection/nuclearInteraction/NaivNuclearInteraction.h"
+#include "PROPOSAL/Constants.h"
+#include "math.h"
+
+using namespace PROPOSAL;
+
+std::array<double, 2>
+stochastic_deflection::NaivNuclearInteraction::CalculateStochasticDeflection(
+        double e_i, double e_f, std::vector<double> const& rnd) const
+{
+    // ------- NuclInt 
+    auto proton_mass = 938.272; // Proton mass in MeV
```

Comment:
For all physical constants, we should use the variables from `PROPOSAL/Constants.h` (in this case, MP for the proton mass) to be consistent throughout the whole project.